kbase-extension/static/kbase/js/widgets/function_output/kbaseFeatureSet.js

```diff
@@ -155,10 +156,15 @@ define (
                                 // every feature we get back here is something in the list
                                 for(var f=0; f<g.features.length; f++) {
                                     var aliases = "None";
-                                    if(g.features[f].aliases) {
-                                        if(g.features[f].aliases.length>0) {
-                                            aliases= g.features[f].aliases.join(', ');
+                                    if (g.features[f].aliases && g.features[f].aliases.length>0){
+                                        console.log(g.features[f].aliases[0]);
+                                        if (g.features[f].aliases[0] instanceof Array){
+                                            g.features[f].aliases = g.features[f].aliases.map(x => x[1])
```

Comment:
missing semicolon, and ES6 syntax probably won't work in all browsers.

Reply:
Looks like map is pretty well supported (I think the use I have is even supported by IE11). Do you still want me to use something else?